src/index.js

```diff
@@ -794,11 +806,44 @@ const members = {
       return this.removeChild(refNode);
     },
   },
+  setAttribute: {
+    value(attrName, attrValue) {
+      if(attrName === 'slot') {
+        this[attrName] = attrValue;
+      }
+
+      if(isSlotNode(this)) {
+        if(attrName === 'name') {
+          this[attrName] = attrValue;
```

Comment:
It won't, because the property setter calls the native `setAttribute`, not our overriden one.